src/settings.rs

```diff
@@ -36,7 +36,8 @@ pub struct CommonSettings {
         clap(
             long,
             value_parser,
-            name = "channel",
+            long = "channel",
```

Comment:
Seems weird to me that there are two definitions of `long`

Reply:
Great catch, thanks